src/cpp/session/modules/SessionRmdNotebook.R

```diff
@@ -557,6 +557,9 @@ assign(".rs.notebookVersion", envir = .rs.toolsEnv(), "1.0")
    if (length(csvData) == 0 || nrow(csvData) == 0)
       return(list())
    
+   # remove ANSI escapes used for color from text
+   csvData$text <- gsub("\033\\[\\d+m", "", csvData$text)
```

Comment:
Don't know if it's possible here, but it would be valid ANSI escape to include multiple numbers (2 or more) separated by semicolons, e.g.: ESC[1;32m is "bold green".

Reply:
Also, it is valid to have **no** digits, ESC[m means "reset to defaults".